app-i18n/translate-shell/translate-shell-0.9.6.9.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Online command-line translator"
+HOMEPAGE="https://www.soimort.org/translate-shell/"
+SRC_URI="https://github.com/soimort/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+curl +bidi test tts"
+
+RDEPEND="
+	app-misc/rlwrap
+	>=sys-apps/gawk-4.0.2
```

Comment:
This version is long gone. Please change this line to `sys-apps/gawk` instead.

Reply:
Thanks @monsieurp, I updated that dependency too, and squashed it into the bump commit.